pkg/volume/cinder/attacher_test.go

```diff
@@ -57,6 +57,24 @@ func TestGetDeviceName_PersistentVolume(t *testing.T) {
 	}
 }
 
+func TestGetDeviceMountPath(t *testing.T) {
+	spec := createVolSpec("cinder-volume-id", false)
```

Comment:
please change `"cinder-volume-id"` to a variable, and use this variable instead of hard coded string below